praw/models/reddit/comment.py

```diff
@@ -12,6 +14,12 @@
 )
 from .redditor import Redditor
 
+_Comment = TypeVar("_Comment")
```

Comment:
Why is this underscored?

Reply:
You cannot use a class name when defining methods of a class. If I define a method in class X that returns a list of X’s, I cannot do List[x] for return so I made a type var called _X instead

Reply:
Of course, that makes sense. I didn't take notice to what file you're editing. Thanks for the explanation.